# HG changeset patch
# User Rik <rik@octave.org>
# Date 1357537949 28800
#      Sun Jan 06 21:52:29 2013 -0800
# Node ID 8e38eac05230dcc75f235218f34ca29912394a40
# Parent  5a4a07a2ec732410a2d1ce47b06a281c60840584
doc: Change object type of dbwhere, dbstack to "Command" in docstring.

* libinterp/interpfcn/debug.cc(Fdbwhere): Change object type to "Command" in
docstring.

* libinterp/interpfcn/debug.cc(Fdbstack): Change object type to "Command" in
docstring.

diff --git a/libinterp/interpfcn/debug.cc b/libinterp/interpfcn/debug.cc
--- a/libinterp/interpfcn/debug.cc
+++ b/libinterp/interpfcn/debug.cc
@@ -696,17 +696,17 @@ A line number, or vector of line numbers
       retval.assign ("line", line);
 
       return octave_value (retval);
     }
 }
 
 DEFUN (dbwhere, , ,
   "-*- texinfo -*-\n\
-@deftypefn {Built-in Function} {} dbwhere ()\n\
+@deftypefn {Command} {} dbwhere\n\
 In debugging mode, report the current file and line number where\n\
 execution is stopped.\n\
 @seealso{dbstatus, dbcont, dbstep, dbup}\n\
 @end deftypefn")
 {
   octave_value retval;
 
   octave_user_code *dbg_fcn = get_user_code ();
@@ -1022,18 +1022,18 @@ do_dbstack (const octave_value_list& arg
 void
 show_octave_dbstack (void)
 {
   do_dbstack (octave_value_list (), 0, std::cerr);
 }
 
 DEFUN (dbstack, args, nargout,
   "-*- texinfo -*-\n\
-@deftypefn  {Built-in Function} {} dbstack ()\n\
-@deftypefnx {Built-in Function} {} dbstack (@var{n})\n\
+@deftypefn  {Command} {} dbstack\n\
+@deftypefnx {Command} {} dbstack @var{n}\n\
 @deftypefnx {Built-in Function} {[@var{stack}, @var{idx}] =} dbstack (@dots{})\n\
 Display or return current debugging function stack information.\n\
 With optional argument @var{n}, omit the @var{n} innermost stack frames.\n\
 \n\
 The optional return argument @var{stack} is a struct array with the\n\
 following fields:\n\
 \n\
 @table @asis\n\
